docs/pages/icons.vue

```diff
@@ -5,79 +5,114 @@
 
     <PageSection title="Overview" anchor="#overview">
       <p>
-        Kolibri products uses a large number of icons that have a special meaning within our ecosystem. They generally draw from <ExternalLink text="Material Icons" href="https://material.io/resources/icons/" />, but may also include custom icons.
+        Kolibri products use a large number of icons that have a special meaning within our ecosystem. They generally draw from <ExternalLink text="Material Icons" href="https://material.io/resources/icons/" />, but may also include custom icons.
       </p>
       <p>
         Importantly, we reference icons by a token specific to the Kolibri Design System rather than Material's names, in order to make token references more meaningful to designers and developers.
       </p>
+    </PageSection>
+
+    <PageSection title="Size and text" anchor="#size-text">
+      <p>
+        Icons should be scaled relative to their surrounding text size. Use the <code>KLabeledIcon</code> component to automatically set a consistent sizing and spacing with associated text:
+      </p>
+      <h3>
+        <KLabeledIcon icon="lesson" label="Header 3" />
+      </h3>
+      <h4>
+        <KLabeledIcon icon="lesson" label="Header 4" />
+      </h4>
+      <h5>
+        <KLabeledIcon icon="lesson" label="Header 5" />
+      </h5>
       <p>
-        Note that many of these icons also are used with  <InternalLink href="colors" text="conventional colors" />
-        in the design system.
+        <KLabeledIcon icon="lesson" label="Paragraph" />
       </p>
+    </PageSection>
 
+    <PageSection title="Color" anchor="#color">
+      <p>
+        When inline with text, icons should be the same color as the text.
+      </p>
+      <p>
+        Note that many of these icons also are used with <InternalLink href="colors" text="conventional colors" /> in the design system. For example, the <InternalLink href="#tokens-coach" text="coach" code /> icon is often shown using the <InternalLink href="/colors#tokens-coachContent" text="coachContent" code /> color, e.g. <KIcon icon="coach" :color="$themeTokens.coachContent" />.
```

Comment:
Should we cover icons that are specific colors all the time? Help needed, completed, incorrect answers, etc all have specific colors. Also when they're used inline, they're not the same color as the text.

Reply:
good points thanks

Reply:
> icons that are specific colors all the time

Perhaps this is something we should bake into `KIcon`? specifying a particular `icon` prop could set a default color, which could be overwritten?

Reply:
Yeah I think that's a good call.

Reply:
updated